config/routes.rb

```diff
@@ -9,4 +9,7 @@
   # Defines the root path route ("/")
   # root "articles#index"
   get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  # get '/users/:user_id/posts/:post_id
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused codes/comments in order to have a clean codebase.